CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 * (07-tendermint) [\#210](https://github.com/cosmos/ibc-go/pull/210) Export all consensus metadata on genesis restarts for tendermint clients.
 * (core) [\#200](https://github.com/cosmos/ibc-go/pull/200) Fixes incorrect export of IBC identifier sequences. Previously, the next identifier sequence for clients/connections/channels was not set during genesis export. This resulted in the next identifiers being generated on the new chain to reuse old identifiers (the sequences began again from 0).
 * (02-client) [\#192](https://github.com/cosmos/ibc-go/pull/192) Fix IBC `query ibc client header` cli command. Support historical queries for query header/node-state commands.
+* (02-client) [\#500](https://github.com/cosmos/ibc-go/pull/500) Fix IBC `update-client proposal` cli command to expect correct number of args.
```

Comment:
You should add this to the `[Unreleased]` section at the top instead, under `Bug Fixes` (you might need to create this section). I just merged a PR to main updating the changelog, so please update your branch first to avoid conflicts.

Reply:
done